spec/support/mock_generator.rb

```diff
@@ -0,0 +1,12 @@
+# Used in generator_example_helpers_spec.rb
+# set_shell_prompt_responses requires a generator w/ a shell
+# as an argument. This class defines that generator w/ a shell.
+require_relative 'mock_shell'
+
+class Generator
+  attr_accessor :shell
+
+  def initialize
+    @shell = Shell.new
```

Comment:
It seems like overkill to create a Shell class in another file just for this one line.  What do you think of doing it all here like below?

```
def initialize
  Shell = Class.new
  @shell = Shell.new
end
```